db/update.go

```diff
@@ -0,0 +1,28 @@
+package db
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+)
+
+// Update - Replaces an existing resource in the repository
+func (h DynamodbDatabase) Update(resource interface{}) error {
+	row, err := dynamodbattribute.MarshalMap(resource)
+
+	if err != nil {
```

Comment:
do we want eventually a shared definition for this simple error handling that can format the response better for the 500 back to the client?